util/lib.py

```
@@ -342,41 +343,69 @@ def load_yaml(filepath):
-    logging.info(f"Preparing {o_path} for dashboard.")
-    
-    callstring = ['robot', 'merge', '-i', o_path]
-    
+def robot_prepare_ontology(
+    o_path: str,
+    o_out_path: str,
+    o_metrics_path: str,
+    base_iris: List[str],
+    make_base: bool,
+    robot_prefixes: Optional[Dict[str, str]] = None,
+    robot_opts="-v",
```

Ruff

Missing type annotation for function argument `robot_opts`

(ANN001)